components/scream/src/share/util/scream_common_physics_functions_impl.hpp

```diff
@@ -384,6 +384,78 @@ void PhysicsFunctions<DeviceT>::calculate_mmr_from_vmr(const MemberType& team,
   });
 }
 
+//+++++++++++PMC
+template<typename DeviceT>
+template<typename ScalarT>
+KOKKOS_INLINE_FUNCTION
+void PhysicsFunctions<DeviceT>::lapse_T_for_psl(const ScalarT& T_ground, const ScalarT& p_ground, const ScalarT& phi_ground,
```

Comment:
This function should only be used by calculate_psl and some unit tests. I wasn't sure whether I should treat it differently (e.g. making it private or something) because of this.